src/Sessions.cc

```diff
@@ -747,7 +745,7 @@ void NetSessions::DoNextPacket(double t, const Packet* pkt, const IP_Hdr* ip_hdr
 		}
 
 	default:
-		Weird(fmt("unknown_protocol_%d", proto), pkt, encapsulation);
+		Weird("unknown_protocol", pkt, encapsulation);
```

Comment:
Was removing the proto from addl intentional here?

Reply:
Yes, intentional -- `net_weird` and `flow_weird` have no "addl".

I'm up for adding an "addl" to those events, but it's awkward at the moment: adding an event parameter breaks user-code, which I don't like (and #262 would solve).  I also don't like the idea of adding new event names like `net_weird_addl` and `flow_weird_addl` just to add more params.

Here's a list of effected net/flow weird events:

- "unknown_mobility_type_%d" -> unknown_mobility_type
- "unknown_routing_type_%d" -> unknown_routing_type
- "unknown_protocol_%d" ->  unknown_protocol
- "unknown_gre_version_%d" -> unknown_gre_version
- "unknown_gre_protocol_%u16" -> unknown_gre_protocol